submissions/nilamop/popup/index.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="uk">
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<link href="css/css.css" rel="stylesheet" type="text/css">
+	<title>Hooli-style Popup</title>
+<body>
+	<div class="menu">
+		<a class="button" href="">Mail</a>
+		<a class="button" href="">Images</a>
+		<a class="button" href="#popup1">
+			<img src="img/popup-button.png" alt="Додатки">
+		</a>
+		<a class="button" href="">
+			<img src="img/bell.png" alt="Сповіщення">
+		</a>
+		<a class="button" href="">
+			<img src="img/favicon.png" alt="Профіль">
+		</a>
+	</div>		
+		<div class="popup" id="popup1">	
+    	<a class="close" href="#">&times;</a>
+			<div class="content">
+
```

Comment:
The content in the popup must be unaccessible from the keyboard. In your case not.